sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/models/SettingSelector.java

```diff
@@ -202,18 +202,18 @@ public SettingSelector setRange(Range range) {
 
     @Override
     public String toString() {
-        String fields;
-        if (ImplUtils.isNullOrEmpty(this.fields)) {
-            fields = "ALL_FIELDS";
+        String selects;
+        if (ImplUtils.isNullOrEmpty(this.selects)) {
+            selects = "SELECT_ALL";
```

Comment:
@conniey Do you remember if `ALL_FIELDS` triggered the service to return something special? Looking at their documentation there is no mention of this and we likely should pass nothing in this case to get the default selection.